omise/controllers/front/payment.php

```diff
@@ -0,0 +1,28 @@
+<?php
+if (! defined('_PS_VERSION_')) {
+    exit();
+}
+
+if (defined('_PS_MODULE_DIR_')) {
```

Comment:
@nimid better to check if `_PS_MODULE_DIR_` is not defined, then, `exit();`? otherwise the code below won't work.